app/models/school.rb

```diff
@@ -0,0 +1,12 @@
+class School < ApplicationRecord
+  has_many :trips
+
+  validates :name,
+    :responsable_name,
+    presence: true
+
+  validates :responsable_contact,
+    presence: true,
+    numericality: true,
```

Comment:
Why does this need to be a number? doesn't it make more sense to be a string?
Also in the database, the column's type is a string.
You might want to allow people to enter "+36-333-..." or "(36) 333 1234" or "333 12345678" or "333.123.4567"

Reply:
Oh, yes, it makes sense, I changed it, thanks!